go/vt/vtctl/grpcvtctldserver/server.go

```diff
@@ -2033,6 +2035,228 @@ func (s *VtctldServer) UpdateCellsAlias(ctx context.Context, req *vtctldatapb.Up
 	}, nil
 }
 
+// ValidateSchemaKeyspace will diff the schema from all the tablets in
```

Comment:
these should follow the comments of the other methods. the actual documentation belongs on the proto definition